### DISCUSSION TITLE
Security Onion SIEM Integrations

### DISCUSSION THREAD

**Discussion by mccroryc** [Security-Onion-Solutions/securityonion#13018] [Unsupported Versions] [answered]
I am just getting started with Security Onion and have been tasked with setting the product up as our SIEM. Would anyone know if there is a list of compatible products that have documentation for data integration. Specifically I would like to integrate Microsoft Defender, Cisco FTD, PF Sense, Tenable network monitor

thank you for your help
Kind regards,
Chris

**Comment by dougburks** [comment, marked as answer]
For pfSense, please see:
https://docs.securityonion.net/en/2.4/pfsense.html

For Microsoft Defender, Cisco FTD, and Tenable, please see:
https://docs.securityonion.net/en/2.4/elastic-fleet.html#integrations
